municipalities/beek-design-tokens/src/component/conduction/navigation.tokens.json

```diff
@@ -0,0 +1,88 @@
+{

```

Comment:
This navbar does not look anything like the navbar on the website
https://www.gemeentebeek.nl/

<img width="805" height="264" alt="image" src="https://github.com/user-attachments/assets/727c8f8a-c38e-46cc-8114-8468da81650c" />